index.html

```diff
@@ -388,6 +380,21 @@ <h2 class="title">Contact me</h2>
             </div>
         </footer>
     </main>
+    <!-- Menu Modal -->
+    <div id="menu-modal" class="menu-modal">
+        <nav>
+            <ul id="nav-list" >
+                <li><a href="#headline">Portoflio</a></li>
+                <li><a href="#about">About</a></li>
+                <li><a href="#contact">Contact</a></li>
+            </ul>
+        </nav>
+        <button id="close-modal" class="close-modal">
+            <span>&times;</span>
+        </button>
+    </div>
```

Comment:
Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![Screenshot_5](https://user-images.githubusercontent.com/69638013/161509772-8d06045b-1c2e-429a-a19f-5a13841832a9.png) |  ![Screenshot_6](https://user-images.githubusercontent.com/69638013/161516152-e4048cd7-5585-4a42-a6b3-b400f40cc6ab.png)

- [ ] Kindly make the background blurry when the mobile navigation is open, this will make the mobile nav text more visible to the users 👍🏿 


- Adding this functionality will make your design good and look exactly like the Figma design, great job so far 💪🏿

Reply:
Thanks 👍🏿